.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+          # Check out the complete branch history
+          fetch-depth: 0
+
+      - name: "Generate release changelog"
+        id: generate-release-changelog
+        # Version 2.4 is available, but creates scrambled output
+        uses: heinrichreimer/github-changelog-generator-action@v2.3
```

Comment:
What is "heinrichreimer" here? External depency? Is that thrustworthy? 🤔

Reply:
It has been working okay the past 5 years. It's open source and my last review of that version was fine.

Reply:
Does not really answer my question what it is. The account *heinrichheimer* has zero repos on GitHub and I could not find it on GitHub actions marketplace yesterday.

Reply:
is this the right repo? https://github.com/janheinrichmerker/action-github-changelog-generator

Reply:
here are som breadcrumbs leading nowhere: https://data.code.gouv.fr/usage/actions/heinrichreimer%2Faction-github-changelog-generator

Reply:
some more... https://github.com/krasnoukhov/homeassistant-tesy/blob/b4aa9684f87d119a1fc416a307df6b167571a5c7/.github/workflows/release.yml#L54

Reply:
Looks like it moved.

Reply:
Okay fine, so then I know now where to look what it does in detail.  Should not stop you from proceeding, I'm just somewhat behind with all this new GitHub action workflow stuff.